scipy/io/matlab/_mio.py

```diff
@@ -77,7 +77,7 @@ def mat_reader_factory(file_name, appendmat=True, **kwargs):
     elif mjv == 1:
         return MatFile5Reader(byte_stream, **kwargs), file_opened
     elif mjv == 2:
-        raise NotImplementedError('Please use HDF reader for matlab v7.3 files,'\
+        raise NotImplementedError('Please use HDF reader for matlab v7.3 files,'
                                   ' e.g. h5py')
```

Comment:
```suggestion
        raise NotImplementedError('Please use HDF reader for matlab v7.3 '
                                  'files, e.g. h5py')
```
The linter didn't mention it before, but the line was too long, too.